src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -293,13 +268,30 @@ object AOIMonitorUtils extends Logging{
     ???
   }
 
-  def queryLastSequence(databaseURI: URI, procName: String): Option[Int] = {
+  def getLastSequence(interval: Interval): Option[Int] = {
+    queryBeginSequence(interval)
+  }
+
+  def getCurrentSequence(augmentedDiffSource: URI, interval: Interval)(
+      implicit spark: SparkSession): Option[Int] = {
+    spark.sparkContext
+      .parallelize(Seq(augmentedDiffSource))
+      .map { uri =>
+        queryBeginSequence(interval)
+      }
+      .collect
+      .apply(0)
+  }
```

Comment:
I want to point out that this function wraps DB accesses in a spark action.  Depending on the environment, the master node may not have the JDBC drivers properly installed, but the executors will.  I've surely encountered this problem on EMR, but I can't recall if such weirdness is also present on ECS.  Keep this in mind when we actually deploy, in case DB accesses start failing.  This will be the pattern to fall back on.

Reply:
Huh. Well that makes sense but I would never have guessed that's why. I assumed that you had just copy pasted these two functions to stub them in without addressing implementation specifics. 

If this becomes an issue, won't it also then be an issue for the `queryAreaOfInterest` call that gets the AOIs from the database?

In any case, I don't plan to make any immediate changes here (aside from perhaps a comment) but noted for later for sure.